tensorflow/python/training/initial_learning_rate_decay.py

```diff
@@ -0,0 +1,190 @@
+# Copyright 2015 Google Inc. All Rights Reserved.
```

Comment:
Hmm...  It is just an option to use exp as base rather than power.  But it
does make the naming a little confusing.  Do you have suggestions?  Thanks.

On Mon, Mar 28, 2016, 1:01 PM Vijay Vasudevan <notifications@github.com>
wrote:

> In tensorflow/python/training/initial_learning_rate_decay.py
> <https://github.com/tensorflow/tensorflow/pull/1512#discussion_r57623768>:
>
> > @@ -0,0 +1,190 @@
> > +# Copyright 2015 Google Inc. All Rights Reserved.
>
> I guess I don't see how exponential_decay is different:
> https://github.com/tensorflow/tensorflow/blob/master/tensorflow/python/training/learning_rate_decay.py#L51
> shows the same type of pattern -- let me know if I'm missing something
> subtle!
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly or view it on GitHub
> <https://github.com/tensorflow/tensorflow/pull/1512/files/8b94a176675164bf99f6d92e3bace6b5fa5b5bc3#r57623768>
>